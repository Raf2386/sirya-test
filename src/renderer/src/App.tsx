import { ReactElement } from 'react'

import electronLogo from './assets/electron.svg'
import Config from './components/config/Config'

const App = ():ReactElement => {
  // const ipcHandle = (): void => window.electron.ipcRenderer.send('ping')

  return (
    <>
      <img alt="logo" className="logo" src={electronLogo} />
      <Config />
    </>
  )
}

export default App
