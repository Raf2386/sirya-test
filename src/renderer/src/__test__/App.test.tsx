import '@testing-library/jest-dom'
import { render } from "@testing-library/react"
import App from "../App"

test('TEST JEST', () => {
  expect(true).toBe(true)
})

test("Renders the main page", () => {
  const { asFragment } = render(<App />)
  expect(asFragment()).toMatchSnapshot()
  console.log(asFragment().firstChild)
  expect(asFragment().querySelector('img')).not.toBeInTheDocument()
  // expect(asFragment().firstChild.querySelector('#config')).toBeInTheDocument();
})
