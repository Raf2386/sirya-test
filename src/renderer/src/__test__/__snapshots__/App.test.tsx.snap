// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the main page 1`] = `
<DocumentFragment>
  <img
    alt="logo"
    class="logo"
    src="[object Object]"
  />
  <div
    id="config"
  >
    <h1>
      CIAO RAFFA !
    </h1>
  </div>
</DocumentFragment>
`;
